services/QuillLMS/client/app/bundles/Staff/components/evidence/llmPromptDatasets/individualTrial/__tests__/trialNotesModal.test.tsx

```diff
@@ -0,0 +1,115 @@
+import * as React from 'react';
+import { render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import TrialNotesModal from '../trialNotesModal';
+import { updateTrial } from '../../../../../utils/evidence/genAIAPIs';
+import { TrialInterface } from '../../../../../interfaces/evidenceInterfaces';
+
+jest.mock('../../../../../utils/evidence/genAIAPIs', () => ({
+  updateTrial: jest.fn(),
+}));
+
+describe('TrialNotesModal', () => {
+  const mockCloseModal = jest.fn();
+  const mockSetTrialAndInvalidateCache = jest.fn();
+  const trial: TrialInterface = {
+    id: 1,
+    number: 123,
+    temperature: 0.7,
+    notes: 'Initial trial note',
+    results: { confusion_matrix: [[1, 2], [3, 4]] },
+    status: 'completed',
+    trial_duration: 60,
+    trial_errors: [],
+    dataset_id: 1,
+    llm_id: 1,
+    llm_prompt_id: 1,
+    optimal_correct: 10,
+    optimal_count: 20,
+    suboptimal_correct: 5,
+    suboptimal_count: 10,
+    average_g_eval_score: 0.8,
+    llm_version: 2,
+    created_at: '2024-11-07T00:00:00Z',
+    updated_at: '2024-11-07T00:00:00Z',
+  };
+
+  const defaultProps = {
+    trial,
+    closeModal: mockCloseModal,
+    setTrialAndInvalidateCache: mockSetTrialAndInvalidateCache
+  }
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('renders the modal with initial trial notes', () => {
+    render(
+      <TrialNotesModal {...defaultProps} />
+    );
+
+    expect(screen.getByText('Trial Notes')).toBeInTheDocument();
+    expect(screen.getByLabelText('Notes')).toHaveValue('Initial trial note');
```

Comment:
Small style thing to take or leave: I'm generally inclined to avoid risks of typo-ing string constants that are defined in the test by referencing them by variable name in `expect`.  I think this works with `expect(...).toHaveValue(trial.notes)`, which makes it explicit that we have this output expectation because of the value of the input.